components/lib/button/Button.vue

```diff
@@ -41,7 +41,7 @@ export default {
             return this.$attrs.disabled || this.$attrs.disabled === '' || this.loading;
         },
         defaultAriaLabel() {
-            return this.label ? this.label + (this.badge ? ' ' + this.badge : '') : this.$attrs['aria-label'];
+            return this.label ? this.label + (this.badge ? ' ' + this.badge : '') : this.$attrs['ariaLabel'];
```

Comment:
Not related to this PR:
I was quite surprised by this behavior. Intuitively, I expected the opposite based on naming.
It should try to use `this.$attrs['ariaLabel']` first and fallback on `this.label` as a default.

```ts
const wrapper = mount(Button, {
  props: {
      label: 'bob',
      aria-label: 'tuna'
  }
});
```
With the current implementation, when we pass both properties, `aria-label` is discarded and label is used for the `aria-label` value 🤔